GAMSPy_integration_example/trnsport_cuopt.ipynb

```
@@ -52,6 +54,40 @@
+   "source": [
+    "import subprocess\n",
+    "from IPython.display import display, HTML\n",
+    "\n",
+    "def check_gpu():\n",
+    "    try:\n",
+    "        output = subprocess.check_output(\"nvidia-smi\", shell=True).decode()\n",
+    "        display(HTML(f\"\"\"\n",
+    "        <div style=\"border:2px solid #4CAF50;padding:10px;border-radius:10px;background:#e8f5e9;\">\n",
+    "            <h3>✅ GPU is enabled</h3>\n",
+    "            <pre>{output.splitlines()[2]}</pre>\n",
+    "        </div>\n",
```

Ruff (0.14.0)

Found useless expression. Either assign it to a variable or remove it.

(B018)